src/dotty/tools/dotc/reporting/Reporter.scala

```diff
@@ -12,44 +12,9 @@ import config.Printers
 import java.lang.System.currentTimeMillis
 import typer.ErrorReporting.DiagnosticString
```

Comment:
I think the weird test errors are because `DiagnosticString` does not exist anymore.